brick/__main__.py

```diff
@@ -246,9 +248,8 @@ def list_(ctx, target):
 
 @cli.command()
 @click.argument("target", default=".")
-@click.argument("skip_previous_steps", default=False)
 @click.pass_context
-def prepare(ctx, target, skip_previous_steps):
+def prepare(ctx, target, skip_previous_steps=None):
```

Comment:
However, this makes sure we can get `skip_previous_steps` passed by `check_recursive`